src/display/canvas.js

```diff
@@ -2105,12 +2105,14 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
       }
 
       var paintWidth = width, paintHeight = height;
-      var tmpCanvasId = 'prescale1';
+      var tmpCanvasId = 'prescale';
```

Comment:
The comment below must be adjusted as well since we're not painting on temporary canvases anymore, but only on one in the end.